DetectorDescription/RegressionTest/src/DDExpandedViewDump.cc

```diff
@@ -22,7 +22,7 @@ void DDExpandedViewDump(ostream& os, DDExpandedView& ex, size_t skip, size_t sto
   bool go(true);
   int count(0);
   bool dotrans(true);
-  if (getenv("DDNOTRANS"))
+  if (std::getenv("DDNOTRANS"))
```

Comment:
Wasn't it an ```std::getenv``` used here? Given that line #1 includes ```<cstdlib>``` and line #17 defines that the std namespace is used - otherwise, the compiler would have complained.

Reply:
@ianna in this specific case it looks redundant, although not incorrect I would say